buidl/test/test_hash.py

```diff
@@ -0,0 +1,9 @@
+from unittest import TestCase
```

Comment:
Good catch. Just realized buidl is also missing tests for all the other `tagged_hash` helper methods (i.e. `hash_aux`)